runtime/onert/core/src/ir/OperandIndexSequence.cc

```diff
@@ -55,6 +55,20 @@ void OperandIndexSequence::replace(const OperandIndex &from, const OperandIndex
   std::replace(_vec.begin(), _vec.end(), from, to);
 }
 
+bool OperandIndexSequence::operator==(const OperandIndexSequence &other) const
+{
+  if (size() != other.size())
+    return false;
+
+  for (uint32_t i = 0; i < size(); ++i)
+  {
+    if (at(i) != other.at(i))
+      return false;
+  }
+
+  return true;
```

Comment:
```suggestion
  return this._vec == other._vec;
```
Using std::vector operator== will work. https://en.cppreference.com/w/cpp/container/vector/operator_cmp

Reply:
I fixed it.